Userland/Libraries/LibCrypto/BigInt/SignedBigInteger.h

```diff
@@ -122,6 +122,7 @@ class SignedBigInteger {
     [[nodiscard]] SignedDivisionResult divided_by(UnsignedBigInteger const& divisor) const;
 
     [[nodiscard]] SignedBigInteger negated_value() const;
+    [[nodiscard]] SignedBigInteger absolute_value() const;
```

Comment:
There already is a method called `unsigned_value()` And you can compare SignedBigInteger with UnsignedBigInteger so maybe that's easier?